package/gluon-mesh-vpn-fastd/luasrc/usr/lib/lua/gluon/mesh-vpn/method/fastd.lua

```diff
@@ -0,0 +1,38 @@
+local uci = require("simple-uci").cursor()
+
+local site = require 'gluon.site'
+local util = require "gluon.util"
+
+local M = {}
+
+function M.public_key()
+	return util.trim(util.exec("/etc/init.d/fastd show_key mesh_vpn"))
+end
+
+function M.enabled()
+	return uci:get_bool("fastd", "mesh_vpn", "enabled")
+end
+
+function M.enable(val)
+	local enabled = '0'
+	if val then
+		enabled = '1'
+	end
+	uci:set("fastd", "mesh_vpn", "enabled", enabled)
```

Comment:
Just pass `true` or `false` to `uci:set`, `simple-uci` will turn that into `'0'` and `'1'`.

Reply:
Missed this. Fixed.